source/connect.txt

```diff
@@ -141,6 +141,59 @@ the shell connected to your selected deployment.
 
    :mongosh:`Perform CRUD Operations in the MongoDB Shell </crud>`
 
+
+.. _vsce-connect-task-atlas-streams:
+
+Create a Connection to Atlas Stream Processing
+----------------------------------------------
+
+.. versionadded:: 1.50
+
+You can use |vsce| to connect to `Atlas Stream Processing 
+<https://www.mongodb.com/products/platform/atlas-stream-processing>`__
+by providing a Stream Processing Interface (SPI) connection string.
+
+For details on how to configure Atlas Stream Processing, see 
+:ref:`manage-spi`. 
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the connections pane
+
+      a. Expand the :guilabel:`Connections` pane in the left navigation 
+         if it is collapsed.
+
+      #. Click the More Actions menu (...) and select
+         :guilabel:`Add MongoDB Connection with Connection String...`
+
+   .. step:: Paste in an SPI connection string
+
+      Once you are connected, the stream connection displays under 
+      the :guilabel:`Connections` pane:
+
+      .. figure:: /images/vsce-stream-connection-example.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+      An example SPI connection string:
+
+      .. code-block::
+
+         mongodb://user1:password1@atlas-stream-xxxxxxxxxxxxxxxxxxxxxxxx-yyyyy.virginia-usa.a.query.mongodb.net/?authSource=admin&readPreference=primary&ssl=true&directConnection=true
+
+   .. step:: Open a new |vsce| playground
+
+      After successfully connecting to your Atlas Stream, on the 
+      :guilabel:`Playgrounds` pane click :guilabel:`Create New Playground`. 
```

Comment:
[suggestion]

Consider re-ordering this like: "After successfully connecting to your Atlas Stream, click :guilabel:`Create New Playground`  on the  :guilabel:`Playgrounds` pane."

Reply:
Updated.